src/SourceGenerators/XamlGenerationTests.Core/TestAttachedPropertyOwner.cs

```diff
@@ -1,13 +1,10 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
+﻿
 using Windows.UI.Xaml;
 
 namespace XamlGenerationTests.Core
 {
-    public static class TestAttachedPropertyOwner
-    {
+	public static class TestAttachedPropertyOwner
```

Comment:
Please preserve the same whitespaces (tabs)

Reply:
Done